src/components/molecules/message/message.html

```diff
@@ -0,0 +1,4 @@
+<div class="message_outer">
+    <div class="message_title">タイトルテキスト</div>
+    <div class="message_content">説明テキスト</div>
+</div>
```

Comment:
```suggestion
<div class="message-outer">
    <div class="message-title">タイトルテキスト</div>
    <div class="message-content">説明テキスト</div>
</div>
```
CSSのクラス名はケバブケースが使われることが多い印象です。
動作として問題ないので、修正は不要です！